### DISCUSSION TITLE
How to change a verified translation string?

### DISCUSSION THREAD

**Discussion by BanCrash** [moneymanagerex/moneymanagerex#4205] [Q&A] [answered]
Hi,
I am going to start translating the remaining strings for Spanish and improve the already translated ones, but I saw that some of them are verified and can't be replaced, even if they are a bad translation (for example, I created an issue for this one: https://crowdin.com/translate/moneymanagerex/470/en-es?filter=basic&value=0#782226 )

How can I replace verified strings?

Regards.

**Comment by vomikan** [comment, marked as answer]
I've just clicked V button
![image](https://user-images.githubusercontent.com/6836805/154149029-06f39669-268d-4121-8de2-85fb282994ef.png)

Then deleted wrong centence:
![image](https://user-images.githubusercontent.com/6836805/154149247-8fdb3204-5aca-4b0d-9672-ef4bca1ff200.png)

**Reply by BanCrash** [reply]
I think you need permission for doing that:
![image](https://user-images.githubusercontent.com/11770760/154149585-09475240-ba05-4333-a912-3d409a373738.png)

In any case thanks for removing the previous one, I will let you know if there are more bad translated then.

**Comment by BanCrash** [comment]
@vomikan I found more incorrect translations. If I manually edit the file will Crowdin get updated too? To avoid disturbing you for each string.

**Reply by vomikan** [reply]
I think so. Only one approved string would be transferred to the app.

**Reply by BanCrash** [reply]
Okay, I will try it then, thanks!
